src/ucp/wireup/select.c

```diff
@@ -534,8 +534,8 @@ static double ucp_wireup_rndv_score_func(const uct_md_attr_t *md_attr,
                                          const uct_iface_attr_t *iface_attr,
                                          const ucp_address_iface_attr_t *remote_iface_attr)
 {
-    /* highest bandwidth */
-    return iface_attr->bandwidth;
+    /* highest bandwidth with lowest overhead */
+    return (iface_attr->bandwidth) * (1 / iface_attr->overhead);
```

Comment:
we need to pick a message size which "examines" the rendezvous protocol to decide which transport is better. maybe a bit larger size would be better, such as 256K